frontend/packages/kubevirt-plugin/integration-tests/tests/vm.wizard.scenario.ts

```diff
@@ -63,9 +63,11 @@ describe('Kubevirt create VM using wizard', () => {
   });
 
   for (const [id, vm] of Object.entries(VMTestCaseIDs)) {
-    const { method } = vm.getData().provisionSource;
+    const method = vm.getData().provisionSource.getValue();
     const specTimeout =
-      method === ProvisionSource.DISK ? CLONE_VM_TIMEOUT_SECS : VM_BOOTUP_TIMEOUT_SECS;
+      method === ProvisionSource.DISK.getDescription()
```

Comment:
can we compare just the provision sources? Also the value will never equal to description right?